operator/v1/types_network.go

```diff
@@ -133,6 +137,20 @@ type OpenShiftSDNConfig struct {
 	UseExternalOpenvswitch *bool `json:"useExternalOpenvswitch,omitempty"`
 }
 
+// KuryrConfig configures the Kuryr-Kubernetes SDN
+type KuryrConfig struct {
+	//TODO(dulek): Remove ports from config once we're 100% sure that there
+	// won't be any conflicts (Kuryr runs with host networking).
+
+	// The port kuryr-daemon will listen for readiness and liveness requests.
+	// +optional
+	DaemonProbesPort *uint16 `json:"daemonProbesPort"`
```

Comment:
Optional properties should be `omitempty`.

Reply:
Done, thanks!